accepted/0000-notification-system-for-cli-updates.md

```diff
@@ -0,0 +1,31 @@
+# Notification system for cli updates
+
+## Summary
+
+A new notification system to warn users about new versions of the **npm cli**.
+
+## Motivation
+
+- Remove an extra http request from the cli
+- Remove dependency on `update-notifier`
+- Original discussion: https://github.com/npm/rfcs/discussions/118
+
+## Rationale and Alternatives
+
+- Do nothing, keep the same system in place
+- More alternatives?
+
+## Implementation
+
+- Set a `user-agent` header in the requests from the **npm cli**
+- Add a check on the registry that notifies user if a greater version of the cli is available
```

Comment:
this needs to take into account the node version; there's very complex logic (https://github.com/nvm-sh/nvm/blob/master/nvm.sh#L150-L269) for figuring out which npm version works on which node version, and it would be pretty bad if npm suggesting updating on a node version where the act of updating completely hosed the entire node install by breaking npm.